### DISCUSSION TITLE
Account was flagged but the Reinstatement Request doesn't help

### DISCUSSION THREAD

**Discussion by ghost** [community/community#121640] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, everyone

This is my first time asking for help here, because at this point I don't know what to do.

## Issue

The issue appeared about 3-4 weeks ago, when my friends couldn't reach my original github account (@turytsia) or my repositories, they had status 404. I didn't really do anything about it for 1-2 days, but as soon as one of my friends couldn't invite me as a contributor to his repository I have decided to contact github support.

At github support I found `Reinstatement Request` option, so I went there and discovered that my account was flagged. I didn't know why, there was no notification or email from github about it, but I created a ticket and waited.

I was waiting for 5-7 days, then my ticket suddenly was closed by github support without any reason or response. The issue, of course, was not fixed so I decided to contact them using their email `support@github.com`, but I couldn't reach it, the email was not found when I was trying to send it from my gmail.

After that I went to github community and I discovered that this issue can be fixed only by `Reinstatement Request`, so I have decided to create one more ticket and provide more information about my problem there. This ticket was also closed, but this time in 3 days with no reason provided (Just like the first one). After my ticket was closed again I decided to create more tickets with different options to get  at least an explanation of why this is happening. The result looks like this (Image below)

<img width="1030" alt="image" src="https://github.com/community/community/assets/167653560/e2b1cdf7-c550-4877-af31-496e80e7ad14">

Additionally, I created one ticket using company's account where I am working and it was closed the same way. Recently, I also discovered label `Spammy` next to my account.

**I would like to know what can I do to resolve this issue, who can I contact? What was done wrong?**

My account @turytsia has 40-45 repositories with a lot of history, so I don't want to copy everything here.

**Comment by manelet** [comment]
same here

**Comment by ebndev** [comment]
Hi @aturytsia,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
